### DISCUSSION TITLE
Arbitrary values for background gradient direction (custom deg)

### DISCUSSION THREAD

**Discussion by chojnicki** [tailwindlabs/tailwindcss#10447] [Ideas]
Based on [docs](https://tailwindcss.com/docs/background-image) there are predefined gradients like `bg-gradient-to-b` but I need other degrees than that. I tried `bg-gradient-[55deg]` and `bg-gradient-to-[55deg]` but nothing. I cannot define that in config because I need to use many values so arbitrary functionality would be way to go. But for now I'm forced to use whole thing as raw CSS in `style=""` which sucks.

Is there any good reason that Tailwind does not support this already? If not - it's my proposal for new feature..

**Comment by wongjn** [comment]
You could use [arbitrary values classes](https://tailwindcss.com/docs/background-image#arbitrary-values) like `bg-[linear-gradient(55deg,var(--tw-gradient-stops))]`.
